src/ucp/core/ucp_context.c

```diff
@@ -723,51 +735,10 @@ const char* ucp_tl_bitmap_str(ucp_context_h context, uint64_t tl_bitmap,
     return str;
 }
 
-static const char* ucp_feature_flag_str(unsigned feature_flag)
-{
-    switch (feature_flag) {
-    case UCP_FEATURE_TAG:
-        return "UCP_FEATURE_TAG";
-    case UCP_FEATURE_RMA:
-        return "UCP_FEATURE_RMA";
-    case UCP_FEATURE_AMO32:
-        return "UCP_FEATURE_AMO32";
-    case UCP_FEATURE_AMO64:
-        return "UCP_FEATURE_AMO64";
-    case UCP_FEATURE_WAKEUP:
-        return "UCP_FEATURE_WAKEUP";
-    case UCP_FEATURE_STREAM:
-        return "UCP_FEATURE_STREAM";
-    case UCP_FEATURE_AM:
-        return "UCP_FEATURE_AM";
-    default:
-        ucs_fatal("Unknown feature flag value %u", feature_flag);
-    }
-}
-
 const char* ucp_feature_flags_str(unsigned feature_flags, char *str,
                                   size_t max_str_len)
 {
-    unsigned i, count;
-    char *p, *endp;
-
-    p    = str;
-    endp = str + max_str_len;
-    count = 0;
-
-    ucs_for_each_bit(i, feature_flags) {
-        ucs_snprintf_zero(p, endp - p, "%s%s", (count == 0) ? "" : "|",
-                          ucp_feature_flag_str(UCS_BIT(i)));
-        count++;
-        p += strlen(p);
-    }
-
-    if (count == 0) {
-        ucs_assert(max_str_len > 0);
-        str[0] = '\0'; /* empty string */
-    }
-
-    return str;
+    return ucs_flags_str(str, max_str_len, &feature_flags, feature_to_str);
```

Comment:
can we remove ucp_feature_flags_str() wrapper and use ucs_flags_str() directly from UCP_CONTEXT_CHECK_FEATURE_FLAGS